bin/resty

```diff
@@ -219,8 +219,8 @@ $inline_lua
 $file_lua
 
                 gen = function()
-                  if inline_gen then inline_gen() end
-                  if file_gen then file_gen() end
+                  if inline_gen then return inline_gen() end
+                  if file_gen then return file_gen() end
```

Comment:
Yeah, it seems it was a bad idea anyways since it is what makes the tests fail because we do not run both at the same time anymore 😅